ding/policy/r2d2.py

```diff
@@ -15,16 +15,16 @@
 
 @POLICY_REGISTRY.register('r2d2')
 class R2D2Policy(Policy):
-    r"""
+    """
     Overview:
         Policy class of R2D2, from paper `Recurrent Experience Replay in Distributed Reinforcement Learning` .
-        R2D2 proposes that several tricks should be used to improve upon DRQN,
-        namely some recurrent experience replay tricks such as burn-in.
+        R2D2 proposes that several tricks should be used to improve upon DRQN, namely some recurrent experience replay \
+        tricks and the burn-in mechanism for off-policy training.
```

Comment:
The R2D2 policy class is inspired by the paper "Recurrent Experience Replay in Distributed Reinforcement Learning". R2D2 suggests the incorporation of several enhancements over DRQN, specifically the application of novel recurrent experience replay strategies and the implementation of a burn-in mechanism for off-policy training.